docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  fxa-oauth-console:
+    build:
+      context: .
+      dockerfile: Dockerfile-build
+    command: node bin/server.js
+    image: mozilla/fxa-oauth-console
+    ports:
+      - "10137:10137"
```

Comment:
see if you can add extra logging and check which one of these is `undefined`: 

https://github.com/mozilla/fxa-oauth-console/blob/892f69ea00799124f77560821b440bd279b9cb4a/lib/routes/oauth.js#L148